documentation/book/proc-manual-rolling-update-kafka.adoc

```diff
@@ -17,20 +17,20 @@ This procedure describes how to manually trigger a rolling update of an existing
 . Find the name of the `StatefulSet` that controls the Kafka pods you want to manually update.
 +
 For example, if your Kafka cluster is named _my-cluster_, the corresponding `StatefulSet` is named _my-cluster-kafka_.
-+
 
 . Annotate a `StatefulSet` resource in {ProductPlatformName}.
 +
 ifdef::Kubernetes[]
 On {KubernetesName}, use `kubectl annotate`:
 [source,shell,subs=+quotes]
 kubectl annotate statefulset _cluster-name_-kafka strimzi.io/manual-rolling-update=true
-endif::Kubernetes[]
 +
+endif::Kubernetes[]
 On {OpenShiftName}, use `oc annotate`:
++
 [source,shell,subs=+quotes]
 oc annotate statefulset _cluster-name_-kafka strimzi.io/manual-rolling-update=true
-+
+
 . Wait for the next reconciliation to occur (every two minutes by default).
 A rolling update of all pods within the annotated `StatefulSet` is triggered, as long as the annotation was detected by the reconciliation process.
 Once the rolling update of all the pods is complete, the annotation is removed from the `StatefulSet`.
```

Comment:
Replace 'Once' with When. (IBM: Use Once only to mean 'one time')

Reply:
Thanks. I've changed this in the Kafka and Zookeeper rolling update modules.